python/foglamp/services/core/api/scheduler.py

```diff
@@ -339,6 +341,100 @@ def _extract_args(data, curr_value):
         raise web.HTTPNotFound(reason=str(ex))
 
 
+async def enable_schedule_with_name(request):
+    """ Enables the schedule for given schedule_name or schedule_id in request payload
+
+    curl -X PUT http://localhost:8081/foglamp/schedule/enable  -d '{"schedule_name": "a schedule name"}'
```

Comment:
should be - curl -X PUT http://localhost:8081/foglamp/schedule/enable  -d '{"schedule_name": "a schedule name"}' -H "Content-Type: application/json"